### PR TITLE
add woo token metadata

### PR DESCRIPTION
**Pull request by kevinwootrade**
# Add WOO token metadata

ERC-20: https://etherscan.io/address/0x4691937a7508860f876c9c0a2a617e7d9e945d4b
NEP-141: https://explorer.mainnet.near.org/accounts/4691937a7508860f876c9c0a2a617e7d9e945d4b.factory.bridge.near
Aurora ERC-20: https://explorer.mainnet.aurora.dev/address/0x99ec8F13b2AFeF5eC49073B9D20df109D25F78c0

### CODE REVIEW

File: tokens/woo.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "ethereum_address": "0x4691937a7508860f876c9c0a2a617e7d9e945d4b",
+    "aurora_address": "0x99ec8f13b2afef5ec49073b9d20df109d25f78c0",
+    "name": "WOO Network",
```
Comment (sept-en) [inline tokens/woo.json:4]: The token name should be the same as the ERC-20 token on Ethereum.

```suggestion
    "name": "Wootrade Network",
```

**Review by sept-en — CHANGES_REQUESTED** [review]
Thanks for the PR. Added a comment that needs to be addressed before merging.

**Review by sept-en — APPROVED** [review]

File: tokens/woo.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+    "ethereum_address": "0x4691937a7508860f876c9c0a2a617e7d9e945d4b",
+    "aurora_address": "0x99ec8f13b2afef5ec49073b9d20df109d25f78c0",
+    "name": "Wootrade Network",
+    "symbol": "WOO",
+    "decimals": 18,
+    "near_address": "4691937a7508860f876c9c0a2a617e7d9e945d4b.factory.bridge.near"
+    "icon": "",
+    "reference": "",
+    "reference_hash": ""
```
Comment (sept-en) [inline tokens/woo.json:10]:
```suggestion
    "ethereum_address": "0x4691937a7508860f876c9c0a2a617e7d9e945d4b",
    "near_address": "4691937a7508860f876c9c0a2a617e7d9e945d4b.factory.bridge.near",
    "aurora_address": "0x99ec8f13b2afef5ec49073b9d20df109d25f78c0",
    "name": "Wootrade Network",
    "symbol": "WOO",
    "decimals": 18,
    "icon": "",
    "reference": "",
    "reference_hash": ""
```

**Comment by kevinwootrade** [comment]
Thanks for merging the PR, when will woo token show up on the rainbowbridge UI? thx

**Pull request merged**